content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -415,3 +415,14 @@ If `/var/lib/kubelet` is being mounted, performing a `kubeadm reset` will effect
 To workaround the issue, re-mount the `/var/lib/kubelet` directory after performing the `kubeadm reset` operation.
 
 This is a regression introduced in kubeadm 1.15. The issue is fixed in 1.20.
+
+## Cannot use the metrics-server securely in a kubeadm cluster
+
+In a kubeadm cluster, the [metrics-server](https://github.com/kubernetes-sigs/metrics-server)
+can be used insecurely by passing the `--kubelet-insecure-tls` to it. In most cases this is not recommended.
```

Comment:
Let's reword this to always recommend TLS for traffic between this add-on and kubelets. Is there ever a case when we don't recommend it?

Reply:
the metrics server docs state the insecure can be used for testing:
> --kubelet-insecure-tls - Do not verify the CA of serving certificates presented by Kubelets. For testing purposes only.

https://github.com/kubernetes-sigs/metrics-server/blob/master/README.md#configuration

i can reword to:
> This is not recommended for production clusters.